src/util/__mocks__/segmentationEvent.mock.js

```diff
@@ -0,0 +1,15 @@
+export default {
+  detail: {
+    element: {},
+    currentPoints: {
+      image: {
+        x: 1,
+        y: 1,
+      },
+    },
+    image: {
+      width: 200,
+      height: 200,
+    },
+  },
+};
```

Comment:
I'm not sure that this needs to be a mock. More of a fake/stub/input

Reply:
This is unused after a cull of useless tests, removing.